### PR TITLE
feat(favicon): Add a magnificent favicon

### PR DESCRIPTION
**Pull request by deuzu**
:+1:  Make it rain :+1:

### CODE REVIEW

**Comment by frenchfal** [comment]
Ceci est la beauté

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "type": "git",
+    "url": "git+https://github.com/deuzu/24chevres.com.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/deuzu/24chevres.com/issues"
+  },
+  "homepage": "https://github.com/deuzu/24chevres.com#readme",
+  "dependencies": {
```
Comment (izeau) [inline package.json]: `devDependencies`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "http-server": "^0.10.0"
```
Comment (izeau) [inline package.json]: Remplace par `live-server`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "24chevres.com",
+  "version": "1.0.0",
+  "description": "Website of 24chevres.com",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (izeau) [inline package.json]: _LAME_

**Review by izeau — CHANGES_REQUESTED** [review]

**Comment by NassK** [comment]
fdp <3

**Comment by deuzu** [comment]
@NassK Democracy has been enforced. Resistance is futile.

**Pull request merged**